### PR TITLE
[DO NOT MERGE] Test ODH 2.9 Release

### PR DESCRIPTION
**Pull request by AjayJagan**
## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: get_all_manifests.sh [hunk 1]
```diff
@@ -6,19 +6,19 @@ GITHUB_URL="https://github.com/"
...
-    ["odh-dashboard"]="opendatahub-io:odh-dashboard:incubation:manifests:dashboard"
-    ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
-    ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
-    ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
-    ["trustyai"]="trustyai-explainability:trustyai-service-operator:release/1.10.2:config:trustyai-service-operator"
-    ["model-mesh"]="opendatahub-io:modelmesh-serving:release-0.11.1:config:model-mesh"
-    ["odh-model-controller"]="opendatahub-io:odh-model-controller:release-0.11.1:config:odh-model-controller"
-    ["kserve"]="opendatahub-io:kserve:release-v0.11.1:config:kserve"
-    ["modelregistry"]="opendatahub-io:model-registry-operator:main:config:model-registry-operator"
+    ["codeflare"]="opendatahub-io:codeflare-operator:v1.1.0-rhoai:config:codeflare"
```
Comment (zdtsw) [inline get_all_manifests.sh:9]: i do not think CFO will use this branch for their release

Comment (VaishnaviHire) [reply get_all_manifests.sh:9]: @astefanutti Can you confirm the release version for codeflare

Comment (astefanutti) [reply get_all_manifests.sh:9]: @VaishnaviHire latest version for CodeFlare is https://github.com/opendatahub-io/codeflare-operator/releases/tag/v1.2.0.

Comment (AjayJagan) [reply get_all_manifests.sh]: https://github.com/opendatahub-io/opendatahub-community/issues/146#issuecomment-1989086281

**Comment by VaishnaviHire** [comment]
Closing as testing completed

**Pull request closed without merging**